### DISCUSSION TITLE
Can't change recipes path

### DISCUSSION THREAD

**Discussion by Qfrijters** [nextcloud/cookbook#1817] [FAQ and user questions] [answered]
Pretty much what the title says, or I might be just beyond stupid (which is entirely possible). I am on nextcloud version 27.1.0  and the latest version of cookbook (from March). I am really new to the cookbook plugin but I couldn't find anything in the documentation (or I just read over it 3 times). The recipe folder is located on a different drive than where I have nextcloud installed. 

Thanks in advance.

https://github.com/nextcloud/cookbook/assets/68586456/c88dd9b5-3fa0-4c75-ac49-ebb017d31ec6

**Comment by christianlupus** [comment]
Hey!

nomally, there should be a button in the lower right corner to confirm the actual selection of the folder:
![Rastergrafik](https://github.com/nextcloud/cookbook/assets/8202922/c8aadc49-1339-494e-9d48-fe2de4b5a59a)

Your video seems to be cropped there so I am not 100% positive it is there. If it was not there, this would be a bug in the dependencies (the dialog is part o the core). Please keep me posted on the result. Give me a hint on your NC version if it was invisible for some reason.

**Comment by Qfrijters** [comment]
I'm on version Nextcloud Hub 6 (27.1.0) , Debian 11 and cookbook version 0.10.2
https://github.com/nextcloud/cookbook/assets/68586456/28b24d7f-4a9e-45cb-bae7-d22f9eccdfa2

**Reply by christianlupus** [reply, marked as answer]
OK, I checked with the team of the server. There is a bug in the legacy code, the cookbook app is using ATM. I will move to the current code which should bring back the button.

**Reply by Qfrijters** [reply]
Thank you very much! Looking forward to the next release :) Have a nice evening and goodnight.

**Reply by christianlupus** [reply]
You too. Hoping for a son release.

**Reply by Qfrijters** [reply]
just noticed that the 27.1.1 update that came out fucks cookbook even more :p 
![image](https://github.com/nextcloud/cookbook/assets/68586456/2f8f2a9e-993c-405f-8aee-ed38622483c3)

**Reply by jfd0** [reply]
seeing the exact same thing as @Qfrijters in the memories and audio player app
